Resources/Prototypes/_RMC14/Entities/Objects/Fun/toys.yml

```
@@ -32,6 +32,35 @@
 - type: entity
   parent: PlushieSharkGrey
   id: RMCPlushieSharkGrey
+  name: Grey shark soft toy
```

i suggest to keep their names lowercase, to keep entity names consistent, but add a suffix instead, it will show up in the spawn menu as "grey shark soft toy [RMC]"
```suggestion
  suffix: RMC
```

---

Fair enough, i wasnt aware of suffix.